examples/hypervisor_cluster_profiles.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright [2020] Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+from pprint import pprint
+
+from config_loader import try_load_from_file
+from hpOneView.oneview_client import OneViewClient
+
+config = {
+    "ip": "<oneview_ip>",
+    "credentials": {
+        "userName": "<username>",
+        "password": "<password>"
+    }
+}
+
+options = {
+    "type": "HypervisorClusterProfileV3",
+    "description": "test cluster",
+    "hypervisorType": "Vmware",
+    "hypervisorClusterSettings": {
+        "type": "Vmware",
+        "drsEnabled": True,
+        "haEnabled": False,
+        "multiNicVMotion": False,
+        "virtualSwitchType": "Standard"
+    },
+    "hypervisorHostProfileTemplate": {
+        "serverProfileTemplateUri": "/rest/server-profile-templates/ab4778bf-f109-4023-9759-7a8b0ac90e36",
```

Comment:
instead of using hardcoded uri, i would suggest you should get the server profile template by name and then get the uri. Something like this: 
profile_templates = oneview_client.server_profile_templates
profile_template_name = "OneView Test Profile Template"
server_template = profile_templates.get_by_name(profile_template_name)
serverProfileTemplateUri=server_template.data["uri"]

Reply:
Done